sdk/cosmos/azure-cosmos/azure/cosmos/aio/_container.py

```diff
@@ -1321,5 +1322,56 @@ async def read_feed_ranges(
 
         feed_ranges = [FeedRangeInternalEpk(Range.PartitionKeyRangeToRange(partitionKeyRange)).to_dict()
                 for partitionKeyRange in partition_key_ranges]
-
         return (feed_range for feed_range in feed_ranges)
+
+    async def get_latest_session_token(self,
+                                       feed_ranges_to_session_tokens: List[Tuple[Dict[str, Any], str]],
+                                       target_feed_range: Dict[str, Any]
+                                       ) -> str:
```

Comment:
The usual formatting for long method signature is:
```python
def method_name(
        self,
        one_long_parameter_name: List[Tuple[Dict[str, Any]]],
        two_long_parameters_names: Optional[str] = None,
) -> None:
    # implementation here
    print("hello")
```

Reply:
changed